examples/navier-stokes/nsplex.c

```diff
@@ -1028,10 +1031,34 @@ int main(int argc, char **argv) {
   user->dt = dt;
 
   // Solve
+  start = MPI_Wtime();
   ierr = TSSolve(ts, Q); CHKERRQ(ierr);
+  ierr = TSGetSolveTime(ts,&ftime); CHKERRQ(ierr);
+  cpu_time_used = MPI_Wtime() - start;
```

Comment:
I'd flip the order of these two lines. Also, you probably want `ierr = PetscBarrier((PetscObject)ts); CHKERRQ(ierr);` between `start = ...` and `TSSolve`.